docs/upgrade/v1-6-0-to-v1-6-1.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade from v1.6.0 to v1.6.1
+title: "Upgrade from v1.6.0 to v1.6.1"
```

Comment:
Consider changing this to 1.6.x -> 1.6.y, as I mentioned in the another comment.